app/stores/AccountStore.js

```diff
@@ -63,9 +65,7 @@ class AccountStore extends BaseStore {
             accountContacts: Immutable.Set(),
             linkedAccounts: Immutable.Set(), // linkedAccounts are accounts for which the user controls the private keys, which are stored in a db with the wallet and automatically loaded every time the app starts
             referralAccount,
-            passwordLogin: accountStorage.get("settings_v4", {
-                passwordLogin: true
-            }).passwordLogin
+            passwordLogin: settings.passwordLogin || true
```

Comment:
This will always return true, even if passwordLogin is set to false.

It should be done like this instead:
```
const storedSettings = ss.get("settings_v4", {passwordLogin: true});
this.state = {
    ...
   passwordLogin: storedSettings.passwordLogin
```